tests/unit_tests/c_wrappers/res/enkf/test_field_export.py

```diff
@@ -32,33 +30,37 @@ def test_field_basics(snake_oil_field_example):
 
 @pytest.mark.skip(reason="pending refactor")
 def test_field_export(snake_oil_field_example):
-    ert = snake_oil_field_example
-    fs_manager = ert.storage_manager
-    ens_config = ert.ensembleConfig()
-    config_node = ens_config["PERMX"]
-    data_node = EnkfNode(config_node)
-    node_id = NodeId(0, 0)
-    fs = fs_manager.current_case
-    data_node.tryLoad(fs, node_id)
+    pass
+    # ert = snake_oil_field_example
+    # fs_manager = ert.storage_manager
+    # ens_config = ert.ensembleConfig()
+    # config_node = ens_config["PERMX"]
+    # data_node = EnkfNode(config_node)
+    # node_id = NodeId(0, 0)
+    # fs = fs_manager.current_case
+    # data_node.tryLoad(fs, node_id)
 
-    data_node.export("export/with/path/PERMX.grdecl")
-    assert os.path.isfile("export/with/path/PERMX.grdecl")
+    # data_node.export("export/with/path/PERMX.grdecl")
+    # assert os.path.isfile("export/with/path/PERMX.grdecl")
 
 
 @pytest.mark.skip(reason="pending refactor")
 def test_field_export_many(snake_oil_field_example):
-    ert = snake_oil_field_example
-    fs_manager = ert.storage_manager
-    ens_config = ert.ensembleConfig()
-    config_node = ens_config["PERMX"]
+    pass
+    # ert = snake_oil_field_example
```

Comment:
Some commented out code here

Reply:
yes just left it there for documenting previous behavior. Can remove the comments :shrug: